x/ethbridge/types/msgs.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/x/ethbridge/common"
+)
+
+// MsgMakeEthBridgeClaim defines a message for creating claims on the ethereum bridge
+type MsgMakeEthBridgeClaim EthBridgeClaim
```

Comment:
as per the comment above this msg should be renamed to something like
```suggestion
type MsgCreateEthBridgeClaim EthBridgeClaim
```

Reply:
fixed: https://github.com/musnit/peggy/commit/586e80686df398a42e305fc14b58f0ccacb81697